### ISSUE TITLE
enabling `astro: true` creates an error with the a11y plugin

### ISSUE THREAD

**Issue by radiosilence** [AndreaPontrandolfo/sheriff#316]
If I enable the astro setting, I get this error:

```[Error - 11:07:45] ConfigError: Config (unnamed): Key "plugins": Cannot redefine plugin "jsx-a11y".```

Brand new project, should be easy to replicate.

**Comment by AndreaPontrandolfo** [comment]
Thanks, I'll look into this.

**Comment by AndreaPontrandolfo** [comment]
So, i wasn't able to reproduce your error, but i run into other errors. More specifically, [this error](https://github.com/ota-meshi/eslint-plugin-astro/issues/447).

I'll open a draft PR for now, but the main fix must come from upstream (the `astro-eslint-plugin`).

**Comment by AndreaPontrandolfo** [comment]
@radiosilence if you can give [eslint-config-sheriff@25.4.0](https://github.com/AndreaPontrandolfo/sheriff/releases/tag/eslint-config-sheriff%4025.4.0) a try, i think it fixed it!
